guides/source/form_helpers.md

```diff
@@ -294,91 +309,91 @@ Which produces the following output:
 
 ```html
 <form action="/people" accept-charset="UTF-8" method="post">
-  <input type="hidden" name="authenticity_token" value="bL13x72pldyDD8bgtkjKQakJCpd4A8JdXGbfksxBDHdf1uC0kCMqe2tvVdUYfidJt0fj3ihC4NxiVHv8GVYxJA==" />
+  <input type="hidden" name="authenticity_token" value="..." autocomplete="off" />
   <input type="text" name="person[name]" id="person_name" />
   <input type="text" name="contact_detail[phone_number]" id="contact_detail_phone_number" />
 </form>
 ```
 
 The object yielded by `fields_for` is a form builder like the one yielded by
-`form_with`.
-
-[`fields_for`]: https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-fields_for
+`form_with`. The `fields_for` helper creates a similar binding but without
+rendering a `<form>` tag. You can learn more about `field_for` in the [API
+docs](https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-fields_for).
 
 ### Relying on Record Identification
 
-The Article model is directly available to users of the application, so - following the best practices for developing with Rails - you should declare it **a resource**:
+When dealing with RESTful resources, calls to `form_with` can be simplified by relying on **record identification**. This means you pass the model instance and have Rails figure out model name, method, and other things. In the example below for creating a new record, both `form_with` have the same generated HTML:
```

Comment:
"both `form_with` have the same generated HTML" was a bit confusing to me.
Maybe:
```suggestion
When dealing with RESTful resources, calls to `form_with` can be simplified by relying on **record identification**. This means you pass the model instance and have Rails figure out model name, method, and other things. In the example below for creating a new record, both calls to `form_with` generate the same HTML:
```